### PR TITLE
Update/add analogwrite alias

### PR DESCRIPTION
**Pull request by edgarsilva**

### CODE REVIEW

**Comment by stewart** [comment]
Can you add `coverage` to the `.gitignore`, and squash these commits?

**Comment by edgarsilva** [comment]
ups, definitely, did not noticed the coverage was pushed. a sec.

File: lib/adaptor.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ Adaptor.prototype.pwmWrite = function(pinNum, value, servo) {
   var pin = this.pwmPins[pinNum];
 
   if (!pin) {
-    pin = new Mraa.Pwm(pinNum);
+    pin = new Mraa.Pwm();
```
Comment (stewart) [inline lib/adaptor.js]: Any reason for this removal?

Comment (edgarsilva) [inline lib/adaptor.js]: non at all, must've been a typo or something, adding it back.

**Pull request merged**